Debulleting the Porch - Guides pages (#557)
nephio-project/porch: docs/content/en/docs/4_tutorials_and_how-tos/working_with_package_revisions/copying-packages.md

>   --workspace v2

Reviewer: This is actually incorrect, copy only takes in 1 argument, that being the full package revision name.

```suggestion
porchctl rpkg copy \
porch-test.my-app.v1 \
--namespace default \
--workspace v2
```

```

> This command creates a new PackageRevision based on `porch-test.my-app.v1` and names the new PackageRevision `my-app` (package name) while using `v2` as the workspace name (must be unique within the package). This PackageRevision starts in `Draft` lifecycle state. The command copies all resources from the source PackageRevision.

Reviewer: ```suggestion
This command creates a new PackageRevision based on `porch-test.my-app.v1` for the package `my-app` with `v2` as the workspace name (must be unique within the package). This PackageRevision starts in `Draft` lifecycle state. The command copies all resources from the source PackageRevision.
```
